package/yast2-packager.spec

```diff
@@ -62,6 +65,9 @@ Requires:       /usr/bin/md5sum
 # .process agent
 Requires:       yast2-core >= 2.16.35
 
+# Storage (methods to check the devicegraph)
+Requires:       libstorage-ng-ruby
```

Comment:
ok, this approach have one disadvantage. In past it is enough to know API and interface of yast2-storage. Now users of API need to learn about yast2-storage API and also libstorage API. So broader API and even more request to have rock solid API of libstorage that are backward compatible.